src/Analytics/Plausible.php

```diff
@@ -0,0 +1,107 @@
+<?php
+/**
+ * Utopia PHP Framework
+ *
+ * @package Analytics
+ * @subpackage Tests
+ *
+ * @link https://github.com/utopia-php/framework
+ * @version 1.0 RC1
+ * @license The MIT License (MIT) <http://www.opensource.org/licenses/mit-license.php>
+ */
+
+namespace Utopia\Analytics;
```

Comment:
Please run this file through the [php-cs-fixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer) to ensure consistency in indentation and formatting.